### PR TITLE
Some improvements to nova script

### PR DESCRIPTION
**Pull request by perl2ruby**
Change shell to bash
Change debug prompt to include line number
Use modern syntax for shell
Display the command about to be run

### CODE REVIEW

File: bin/nova [hunk 1]
```diff
@@ -1,15 +1,26 @@
-#!/bin/sh
+#!/usr/bin/env bash
 #
 # Runs nova
 #
 
+ACTUAL_USER=$(id -un)
+EXPECTED_USER='root'
+PROGRAM_NAME="${0##*/}"
```
Comment (grahamar) [inline bin/nova]: Unused variable?

Comment (perl2ruby) [inline bin/nova]: good point. Have used the program name in the error message when the user running the program is not root

**Comment by gheine** [comment]
Re changing shell from sh to bash: That may not be available on all incantations of linux. Alpine foe example doesn't come with bash preinstalled.

**Comment by grahamar** [comment]
@gheine The script starts a docker container to run nova, docker doesn't come pre-installed on any linux distributions AFAIK :)

For it's purpose, I think it's ok...

**Comment by gheine** [comment]
Ah fair enough, I was thinking this is the nova command line script that would also be run inside docker containers (e.g. for nova stash get).

**Pull request merged**